asistente_ladm_col/logic/quality/quality_rule_engine.py

```diff
@@ -77,18 +83,24 @@ def validate_quality_rules(self):
         else:
             self.logger.warning(__name__, QCoreApplication.translate("QualityRuleEngine", "No rules to validate!"))
 
+        return res
+
     @_log_quality_rule_validations
     def __validate_quality_rule(self, rule_key, layers, rule_name):
         """
         Intermediate function to log quality rule execution.
 
         :param rule_key: rule key
         :param rule_name: Rule name (needed for the logging decorator)
-        :return: tuple (msg, level), where level indicates whether the rule was successful,
-                 couldn't be validated (warning), or was not successful (critical)
+        :return: triple (msg, level, list_of_error_layers), where level indicates whether the rule was successful
```

Comment:
What do you think about encapsulating all arguments in one object? If we need to send more information in the future, the change would be minor.

Reply:
Any example of 'more information'?

Reply:
I can't think of one at the moment but one may come up, just as it happened with list_of_error_layers. At this moment it is not necessary to make the change, but if it happens we can take into account the suggestion.